docs/quality-assurance-and-test.md

```diff
@@ -4,43 +4,147 @@ Welcome to the introduction on how we are working with Quality Assurance and Tes
 
 ## Quality Statement
 
-The content of this section will be overall concepts of thinking and working with quality activities in Green Energy Hub.
+As the founder of this project the danish TSO, Energinet, and Microsoft has together set the bar for the Green Energy Hub to be the best big data handling solution for green energy markets worldwide – and with this statement we all agree to work towards giving the world access to a maintainable, secure, configurable, and fast big data system.  
+We consider every contribution to be thought through, described, peer reviewed and tested before accepted into our product portfolio.
+This is a common responsibility, and are you, as a contributor, not able to meet these guidelines by yourself, please reach out to the community for assistance, input or help towards ensuring your contribution lives up to this quality statement.
 
 ## Quality Risk
 
-Building a product always gives reason to consider product risk areas and the possible costs of these risks elements. In this section we would like to elaborate on how we are working with these considerations, and give inspration to how practically address this.
+Building a product always gives reason to consider product risk areas and the possible costs of these risks elements. In this section we would like to elaborate on how we are working with these considerations, and give inspration on how to practically address this.
+
+In a project like this product risk contains two overall questions
+
+1. *Are we building the right product?*
+2. *Are we building the product right?*
+
+In regards to the first question - we want to be certain that the solution is delivering us the value we need. This question can be answered by evaulating on the quality characteristics of the product. Do the functionality reach the expected level of functionality, security, performance, maintainability, usability etc. Here it is important to have all the relevant considerations outlined before starting to implement. What is the value, the targets, the expectations and the formal requirements for this product.
+
+Writing acceptance criteria and/or use case scenarios before starting to develop is a great way to ensure that all considerations are taken into account during the development phase. But don't forget to try and reason for the costs of not delivering to target or expectation.
+
+Building the right product has in earlier times been proven by a post-developing test - We want to have this mitigation before implementing, as a way of ensuring alignment of expectations. Thus the knowlegde and the learnings of this risk evaluation could also be used for automating regression test within the pipeline so we remember to re-visit our acceptance criteria throughout the delivery pipeline.
```

Comment:
Like this?

" Building the right product has in earlier times been proven by a post-developing test. We want to  mitigate risks during implementation, so that we are working with aligned expectations and relevant measures and are able to avoid large-scaled testing bottle necks before deployment. The knowlegde and the learnings of risk evaluation could also be used for inspiring automated regression tests within a pipeline setup."